css-text-4/Overview.bs

```diff
@@ -8153,6 +8149,19 @@ Character Class Spacing: the 'text-spacing' property</h3>
 
 	Issue: It was requested to add a value for doubling the space after periods.
 
+<h4 id="inter-script-spacing">
+Inter-script Spacing</h4>
+
+	The ''ideograph-alpha'' and ''ideograph-numeric'' values
+	introduce spacing at the boundary between particular classes of characters
+	when they are directly adjoining on a line,
+	i.e. without any intervening non-zero [=margin=], [=border=], or [=padding=]
+	or intervening characters (such as a quotation mark or a space).
+
+	Note: Spacing conventions vary, but values typically range from 1/4ic to as low as 1/8ic,
+	with 1/4ic being particularly common in monospace contexts
+	and 1/6ic being more common in proportional typesetting.
```

Comment:
I'd go with "1/6ic or thinner" rather than "1/6ic". Then 1/8 value that CSS ends up picking is on the thinner side, but not outside of common practice.